src/app/public/modules/datepicker/daypicker.component.html

```diff
@@ -16,7 +16,10 @@
           <sky-icon icon="chevron-left"></sky-icon>
         </button>
       </th>
-      <th [attr.colspan]="5 + (datepicker.showWeeks ? 1 : 0)" scope="col">
+      <th
+        scope="col"
+        [attr.colspan]="5"
```

Comment:
We don't need the `showWeeks` logic anymore? 
```
[attr.colspan]="5 + (datepicker.showWeeks ? 1 : 0)"
```

Reply:
`showWeeks` didn't exist 😕 I only noticed because the build started throwing template errors.